scripts/abins/io.py

```diff
@@ -35,7 +35,7 @@ class IO(BaseModel):
     setting: str = ""
     autoconvolution: int = 10
     temperature: float = None
-    cache_directory: Path
+    cache_directory: Path | None
```

Comment:
This mistake was flagged by mypy while checking annotations in `abinitioloader`.